tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,321 @@
+---
+title: TLS connection to Dedicated Tier
+summary: Introduce TLS connection in TiDB Dedicated Tier.
+---
+
+# TLS connection to Dedicated Tier
+
+On TiDB Cloud, TLS connections are one of the basic security practices for connecting to Dedicated Tier clusters. We support you to configure multiple TLS connections from client, application and development tools to protect your data transmission security.
+
+Each Dedicated Tier cluster server TLS certificate is hosted on [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/), ACM Private CA keys are stored securely in AWS managed hardware security modules (HSMs) that adhere to [FIPS 140-2 Level 3](https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3139) security standards in order to protect your TLS private keys when Dedicated Tier request AWS ACM to create it.
+
+> **Note:**
+>
+> Best practices for network access security include three steps:
+> Step1: Set up [IP Access List](/tidb-cloud/configure-ip-access-list.md)
+> Step2: Set up [Private Link](/tidb-cloud/set-up-private-endpoint-connections.md)
+> Step3: Set up TLS Connection
+
+## Prerequisites
+
+Before connecting to your dedicated cluster, Log in to TiDB Cloud via [Password Authentication](/tidb-cloud-password-authentication.md) or [SSO Authentication](/tidb-cloud/tidb-cloud-sso-authentication.md),and then [Create a TiDB Cloud Serveless Tier cluster](/tidb-cloud/tidb-cloud-quickstart.md).
+
+## Secure connection to a Dedicated Tier cluster
+
+In the [TiDB Cloud console](https://tidbcloud.com/), you can get examples of different connection methods and connect to your Dedicated Tier cluster as follows:
+
+1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, and then click the name of your Dedicated Tier cluster to go to its overview page.
+
+2. Click **Connect** in the upper-right corner. A dialog named standard connection tap is displayed.
+
+3. In this dialog, you should setup three steps:
+   - Step 1：Create traffic filter
+   - Step 2：Download TiDB cluster CA
+   - Step 3：Connect with an SQL client
+
+4. In the Step1：Create traffic filter
+
+Specify the traffic allowed to connect the cluster. Traffic matching any filter is allowed, and all other traffic is denied. For detailed configuration, see: Configure your [IP Access List](/tidb-cloud/configure-ip-access-list.md)
+
+5. In the Step 2：Download TiDB cluster CA
+
+You should click **"**Download TiDB Cluster CA** to download it locally for client TLS configuration.Because of verifying the TiDB cluster CA can ensure the TLS connection is secure and reliable.
```

Comment:
```suggestion
You should click **"**Download TiDB Cluster CA** to download it locally for client TLS configuration. Because of verifying the TiDB cluster CA can ensure the TLS connection is secure and reliable.
```